angular/scripts/linkThemes.sh

```diff
@@ -17,12 +17,24 @@ echo -e "${GREEN}Done${NC}"
 
 echo -en "${BLUE}Copying angular themes into node_modules...${NC}";
 # Showcase demo
-cp -r ./package.json ./demos/showcase/node_modules/@pxblue/angular-themes/package.json
-cp -r ./*.scss ./demos/showcase/node_modules/@pxblue/angular-themes
+cp -r ./demos/showcase/package.json ./demos/showcase/node_modules/@pxblue/angular-themes/package.json
+cp -r ./_blueTheme.scss ./demos/showcase/node_modules/@pxblue/angular-themes/_blueTheme.scss
+cp -r ./_darkTheme.scss ./demos/showcase/node_modules/@pxblue/angular-themes/_darkTheme.scss
+cp -r ./_fonts.scss ./demos/showcase/node_modules/@pxblue/angular-themes/_fonts.scss
+cp -r ./_margins.scss ./demos/showcase/node_modules/@pxblue/angular-themes/_margins.scss
+cp -r ./_typography.scss ./demos/showcase/node_modules/@pxblue/angular-themes/_typography.scss
+cp -r ./_pxb-component-theme.scss ./demos/showcase/node_modules/@pxblue/angular-themes/_pxb-component-theme.scss
+cp -r ./theme.scss ./demos/showcase/node_modules/@pxblue/angular-themes/theme.scss
```

Comment:
What's the reason for going back to the long-form version of the copy script?

Reply:
I think I made this branch using an older version of dev, good catch!  This has been reverted to using the condensed script.